tools/sql_extraction/src/test/kotlin/PythonAnalysisTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.google.cloud.sqlecosystem.sqlextraction
+
+import com.google.cloud.sqlecosystem.sqlextraction.output.QueryUsages
+import org.antlr.v4.runtime.CharStreams
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class PythonAnalysisTest {
+    @Test
+    fun `simple string return`() {
+        val program = """
+def test():
+    return "test"
+"""
+
+        val result = analyze(program)
+
+        assertCorrect(result, "test")
+    }
+
+    @Test
+    fun `simple method argument`() {
+        val program = """
+import something
+something.some_method("test")
+"""
+
+        val result = analyze(program)
+
+        assertCorrect(result, "test")
+    }
+
+    @Test
+    fun `literal concatenation`() {
+        val program = """
+import something
+something.some_method("hello" + "world")
+"""
+
+        val result = analyze(program)
+
+        assertCorrect(result, "(helloworld)")
+    }
+
+    @Test
+    fun `variable assignment and usage`() {
+        val program = """
+import something
+a = "test"
+something.some_method(a)
+"""
+
+        val result = analyze(program)
+
+        assertCorrect(result, "test")
+    }
+
```

Comment:
Does the analyzer support multi-line strings?  If so, please add a unit test. If not, pls add support in another PR.

Reply:
Good idea! I'll add tests for them